plugins/woocommerce-admin/client/marketplace/components/content/content.tsx

```diff
@@ -28,14 +28,46 @@ import ConnectNotice from '~/marketplace/components/connect-notice/connect-notic
 export default function Content(): JSX.Element {
 	const marketplaceContextValue = useContext( MarketplaceContext );
 	const [ products, setProducts ] = useState< Product[] >( [] );
-	const { setIsLoading, selectedTab } = marketplaceContextValue;
+	const { setIsLoading, selectedTab, setHasBusinessServices } =
+		marketplaceContextValue;
 	const query = useQuery();
 
+	// On initial load of the in-app marketplace, fetch extensions, themes and business services
+	// and check if there are any business services available on WCCOM
+	useEffect( () => {
+		const abortController = new AbortController();
+		const categories = [ '', 'themes', 'business-services' ];
+
+		categories.forEach( ( category: string ) => {
+			const params = new URLSearchParams();
+			if ( category !== '' ) {
+				params.append( 'category', category );
+			}
+
+			const wccomSettings = getAdminSetting( 'wccomHelper', false );
+			if ( wccomSettings.storeCountry ) {
+				params.append( 'country', wccomSettings.storeCountry );
+			}
+
+			fetchSearchResults( params, abortController.signal ).then(
+				( productList ) => {
+					if ( category === 'business-services' ) {
+						setHasBusinessServices( productList.length > 0 );
+					}
+				}
+			);
+		} );
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Seems like the Search API  is called twice if i am on that category page
![Screenshot 2024-05-08 at 3 21 31 PM](https://github.com/woocommerce/woocommerce/assets/11981912/969fdf31-8a01-45ab-9ca5-18008051bf57)

Reply:
Yeah, it happens on other tabs as well. It's pre-existing behaviour, but I can check if there's an easy way to prevent it.

Reply:
may be this useEffect need to filter if we are already on that tab page, seems like the useEffect below this one fetches data for current tab?

Reply:
It's pre-existing, so it has been happening even before I'd added this new useEffect hook to to component. And the new hook only runs once, on initial load of the page. It's not triggered by further state changes, like switching tabs.

So whatever is causing the duplicate calls is in the original (now second) useEffect hook.